src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import dukeclass.Parser;
+import dukeclass.Storage;
+import dukeclass.TaskList;
+import dukeclass.Ui;
+
+/**
+ * Encapsulates a chat robot that you can chat with to set tasks for yourself.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    /**
+     * Duke to be started, create files for storage, run commands and write to files.
```

Comment:
Perhaps this could be rewritten to be in line with the coding standard?

> In method header comments, the first sentence should start in the form `Returns ...`, `Sends ...`, `Adds ...` (not `Return` or `Returnning` etc.)